kubernetes/core/src/main/kotlin/com/google/kubernetes/tools/core/kubectl/KubectlExecutorService.kt

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.kubernetes.tools.core.kubectl
+
+import com.intellij.execution.ExecutionException
+import com.google.cloud.tools.intellij.analytics.UsageTrackerService
+import com.google.common.annotations.VisibleForTesting
+import com.google.kubernetes.tools.core.util.CoreBundle.message
+import com.intellij.openapi.components.ServiceManager
+import java.io.BufferedReader
+import java.io.InputStreamReader
+import java.util.concurrent.TimeUnit
+
+const val KUBECTL_FAIL = "kubectl.fail"
+const val METADATA_ERROR_MESSAGE_KEY = "error.message"
+
+class KubectlExecutorService {
```

Comment:
A class level comment is generally very useful when looking at the class for first time, this class very general purpose is a good comment.